src/java/org/apache/cassandra/repair/autorepair/AutoRepair.java

```diff
@@ -203,149 +206,142 @@ public void repair(AutoRepairConfig.RepairType repairType, long millisToWait)
                     }
 
                     repairState.setRepairKeyspaceCount(repairState.getRepairKeyspaceCount() + 1);
-                    List<String> tablesToBeRepaired = retrieveTablesToBeRepaired(keyspace, repairType, repairState);
```

Comment:
Changes look bigger than they actually are, nesting of some logic moved around now that things are done `RepairAssignment`-wise instead of table-wise.